### ISSUE TITLE
Usability testing NVDA (screen reader)

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#3738]
Test the Conditional Logic screens in Windows NVDA. Is it usable? Anything obviously broke?

**Comment by thiessenp-cds** [comment]
Keynav initial testing:
- Tabs can navigate left-right keys (but not up-down). Cannot activate tab w/space/enter
- Questions Tree nav, easy to get into a state where arrow keys stop responding? I think this is my computer lagging though.  (try again later)
